pennylane/circuit_graph.py

```diff
@@ -155,11 +155,9 @@ def __init__(self, ops, variable_deps):
         self.num_wires = 0
         """int: number of wires the circuit contains"""
         for k, op in enumerate(ops):
-            self.num_wires = max(self.num_wires, max(list(_flatten(op.wires))) + 1)
+            self.num_wires = max(self.num_wires, max(op.wires.tolist()) + 1)
```

Comment:
@mariaschuld might be worth putting a todo here. Looks like this logic is implicitly assuming the last index is related to the number of wires

Reply:
Yes, I have it on my radar. The ``tolist()`` serve as implicit ``TODO``s...